alacritty.yml

```diff
@@ -481,9 +481,11 @@ key_bindings:
   - { key: PageDown,                chars: "\x1b[6~"                         }
   - { key: Tab,      mods: Shift,   chars: "\x1b[Z"                          }
   - { key: Back,                    chars: "\x7f"                            }
+  - { key: Back,     mods: Control, chars: "\x17"                            }
   - { key: Back,     mods: Alt,     chars: "\x1b\x7f"                        }
   - { key: Insert,                  chars: "\x1b[2~"                         }
   - { key: Delete,                  chars: "\x1b[3~"                         }
+  - { key: Delete,   mods: Control, chars: "\x1bD"                           }
```

Comment:
The delete escape sequence should be `\x1b[3;5~`. Additionally an action for Shift+Delete should be added (`\x1b[3;2~`).